numpy/f2py/tests/src/cli/hi77.f

```diff
@@ -0,0 +1,3 @@
+      SUBROUTINE HI
```

Comment:
These could probably use a renaming to `hello_world77.f` or some such.

Reply:
Should the other be renamed similarly? We currently have `hi77.f` and `hiworld.f90`. It'd be good to have a template name format and redo all of them at once, all the test sources are oddly named.

Reply:
> all the test sources are oddly named.

Having consistent names would be nice. It isn't a priority, but feel free to do it. It is currently hard to figure out what all the tests do. What ideas do you have for a  "template name format"?

Reply:
I think using the test name or after the refactor the submodule name so:
- `cli_hello_world.f77`
- `cli_hello_world.f90`
- `frontend_no_space` (instead of `gh15035.f`)

Reply:
I agree, avoiding names like hi and blah seems like a good idea in general. Not a blocker though.